Server/routes/Mobile/utils.js

```diff
@@ -0,0 +1,36 @@
+function formatDate(date) {
```

Comment:
no coverage directly on this file (the one on the branch come from the test on the invoice route)